server/controllers/user.js

```diff
@@ -42,14 +44,56 @@ async function login(req, res) {
   }
 
   if (queryCAS(userPseudo, userPassword)) {
+    const admin = await isUserAdmin(userPseudo);
+    const refreshToken = await Tokens.createRefreshToken(userPseudo, admin);
+    const accessToken = Tokens.createAccessToken(refreshToken);
+
     res.sendResponse(200, {
-      pseudo: userPseudo,
-      token: jwt.sign({ pseudo: userPseudo }, process.env.TOKEN_PRIVATE_KEY),
+      user: userPseudo,
+      accessToken,
+      refreshToken,
+      admin,
```

Comment:
Can we rename this as `isAdmin` across the codebase?

Reply:
I will !